app/src/main/java/com/example/themoviedb/feature07/MoviesRepository.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.themoviedb.feature07
+
+import com.example.themoviedb.feature07.model.PopularMovieModel
+import io.reactivex.Observable
+
+interface MoviesRepository {
+    fun getData(): Observable<PopularMovieModel>
```

Comment:
I think a more descriptive method name would be better. Something like getPopularMovies(). getData() is very generic